src/components/posts/PostForm.jsx

```diff
@@ -1,20 +1,91 @@
 /** @jsx createVNode */
 import { createVNode } from "../../lib";
+import { globalStore } from "../../stores";
+
+function updateNewPost(newPostContent) {
+  // console.log('new post');
+  // console.log(newPostContent);
+  // console.log(globalStore.getState().currentUser.username)
+
+  const currentState = globalStore.getState();
+
+  const newId =
+    currentState.posts.length > 0
+      ? Math.max(...currentState.posts.map((post) => post.id)) + 1
+      : 1; // 기존에 게시물이 없으면 id는 1부터 시작
+
+  const newPost = {
+    id: newId,
+    author: globalStore.getState().currentUser.username, // 현재 로그인한 아이디명
```

Comment:
p3;
위에서 선언한 `currentState` 사용해도 좋을 것 같아요
```suggestion
    author: currentState.currentUser.username, // 현재 로그인한 아이디명
```